boa_ast/src/statement/mod.rs

```diff
@@ -226,69 +225,20 @@ impl Statement {
         }
     }
 
-    /// Returns true if the node contains a identifier reference named 'arguments'.
+    #[inline]
```

Comment:
We usually prefer to have attributes after comments, since they are part of the definition. Is there a reason to change it here?

Reply:
Nope, fixed in the next commit